python/numpy/introduction_to_numpy.ipynb

```diff
@@ -1457,15 +1631,14 @@
     "\n",
     "Select elements as with any other Python sequence.\n",
     "\n",
-    "* Indexing starts at `0` for each array dimension\n",
+    "* Indexing starts at `0` for each array dimension®\n",
```

Comment:
not sure I have the meaning of '®'. Please remind it ?

Reply:
Removed.